.gitignore

```diff
@@ -21,7 +21,9 @@ debian/pylint
 .coverage
 .coverage.*
 .idea
+.vscode
```

Comment:
These should go into your `.global-gitignore` instead since they are not related to the project itself but rather to your dev environment.

Reply:
I would argue that it is project specific.  Many projects I have worked on, the .vscode/settings.json file is committed to share settings between devs.  In github's semi-official .gitignore files they recommend committing this file.  It is also consistent with the current .gitignore since .idea is in there as well.  So I don't want to put it in my .global-gitignore as it would break other projects, but I can remove it from this .gitignore if still needed.

Reply:
Fair enough, evidently we did not enforce this rule all the time in the past. Could prevent someone committing it by mistake too.